### PR TITLE
Add tests for remove-user with a rolebinding name

### LINKED ISSUE #18582: Need test for remove-user with invalid role binding name
See https://github.com/openshift/origin/pull/18550#discussion_r167689447

### PR DESCRIPTION
**Pull request by simo5**
Fixes #18582

### CODE REVIEW

**Comment by simo5** [comment]
@enj PTAL

**Comment by enj** [comment]
@simo5 can you also add a test like my example from https://github.com/openshift/origin/pull/18102#discussion_r166825801

File: test/cmd/policy.sh [hunk 1]
```diff
@@ -54,6 +54,10 @@ os::cmd::expect_success_and_text 'oc policy add-role-to-group cluster-admin --ro
 os::cmd::expect_success_and_text 'oc policy add-role-to-user --rolebinding-name cluster-admin cluster-admin system:no-user' 'role "cluster-admin" added: "system:no-user"'
 os::cmd::expect_success 'oc get rolebinding/cluster-admin --no-headers'
 os::cmd::expect_success_and_text 'oc get rolebinding/cluster-admin --no-headers' 'system:no-user'
+# Test failure to mix and mismatch
+os::cmd::expect_success 'oc create rolebinding mismatch --clusterrole=cluster-admin --user=system:user'
+os::cmd::expect_failure 'oc policy remove-user --rolebinding-name mismatch system:no-user'
```
Comment (enj) [inline test/cmd/policy.sh]: os::cmd::expect_failure_and_text with specific failure

**Comment by simo5** [comment]
I could go the whole project creation route like your example, but it's a ton of test overhead for no additional value.

**Comment by simo5** [comment]
And of course there is no --rolebinding-name option on this command ... FAIL! :-)

**Comment by simo5** [comment]
Overhauled the test to use a project and checks for rolebinding mismatches

**Pull request merged**